diff --git a/tmux.h b/tmux.h
index 071d9d29..0ff33cd3 100644
--- a/tmux.h
+++ b/tmux.h
@@ -19,6 +19,7 @@
 #ifndef TMUX_H
 #define TMUX_H
 
+#define environ environ_replace
 #include <sys/time.h>
 #include <sys/uio.h>
 
@@ -31,12 +32,18 @@
 #ifdef HAVE_UTEMPTER
 #include <utempter.h>
 #endif
+#include <stdlib.h>
+#undef environ
 
 #include "compat.h"
 #include "tmux-protocol.h"
 #include "xmalloc.h"
 
+#ifdef __MVS__
+char   **environ;
+#else
 extern char   **environ;
+#endif
 
 struct args;
 struct args_command_state;
