diff --git a/tty-keys.c b/tty-keys.c
index 7725459..d2a7f3b 100644
--- a/tty-keys.c
+++ b/tty-keys.c
@@ -909,7 +909,7 @@ first_key:
 	 * used. termios should have a better idea.
 	 */
 	bspace = tty->tio.c_cc[VERASE];
-	if (bspace != _POSIX_VDISABLE && key == bspace) {
+	if (/* bspace != _POSIX_VDISABLE && */ key == bspace) {
 		log_debug("%s: key %#llx is backspace", c->name, key);
 		key = KEYC_BSPACE;
 	}
@@ -1089,7 +1089,7 @@ tty_keys_extended_key(struct tty *tty, const char *buf, size_t len,
 
 	/* Store the key. */
 	bspace = tty->tio.c_cc[VERASE];
-	if (bspace != _POSIX_VDISABLE && number == bspace)
+	if (/*bspace != _POSIX_VDISABLE &&*/ number == bspace)
 		nkey = KEYC_BSPACE;
 	else
 		nkey = number;
