diff --git a/tty-keys.c b/tty-keys.c
index d3a93f1..404fc55 100644
--- a/tty-keys.c
+++ b/tty-keys.c
@@ -924,7 +924,7 @@ first_key:
 	 * used. termios should have a better idea.
 	 */
 	bspace = tty->tio.c_cc[VERASE];
-	if (bspace != _POSIX_VDISABLE) {
+	if (1) {
 		if (key == bspace) {
 			log_debug("%s: key %#llx is BSpace", c->name, key);
 			key = KEYC_BSPACE;
@@ -1117,7 +1117,7 @@ tty_keys_extended_key(struct tty *tty, const char *buf, size_t len,
 
 	/* Store the key. */
 	bspace = tty->tio.c_cc[VERASE];
-	if (bspace != _POSIX_VDISABLE && number == bspace)
+	if (/*bspace != _POSIX_VDISABLE &&*/ number == bspace)
 		nkey = KEYC_BSPACE;
 	else
 		nkey = number;
